etna/transforms/encoders/categorical.py

```diff
@@ -0,0 +1,235 @@
+import warnings
+from enum import Enum
+from typing import Optional
+
+import numpy as np
+import pandas as pd
+from sklearn import preprocessing
+from sklearn.utils._encode import _check_unknown
+from sklearn.utils._encode import _encode
+
+from etna.transforms.base import PerSegmentWrapper
+from etna.transforms.base import Transform
+
+
+class ImputerMode(str, Enum):
+    """Enum for different imputation strategy."""
+
+    new_value = "new_value"
+    mean = "mean"
+    none = "none"
+
+
+class _LabelEncoder(preprocessing.LabelEncoder):
+    def transform(self, y: pd.Series, strategy: str):
+        diff = _check_unknown(y, known_values=self.classes_)
+
+        index = np.where(np.isin(y, diff))[0]
+
+        encoded = _encode(y, uniques=self.classes_, check_unknown=False).astype(float)
+
+        if strategy == "None":
```

Comment:
Threre is no such value as "None", only "none", may be better to check this using Enum type, not manually.